receiver/tlscheckreceiver/metadata.yaml

```diff
@@ -8,11 +8,10 @@ status:
   codeowners:
```

Comment:
It's technically the line above this one but github won't let me target it - this shows `distrbutions: [config]` when it's not yet in any distribution..

Reply:
Since it was already this way in the previous wireframe PR I'm fine with letting this go and just having the followup PR be soon :tm: